pkg/api/controller.go

```diff
@@ -1885,10 +1884,21 @@ func (c *Controller) GetStorageConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	storageCfg, _ := c.getStorageConfig(config.SingleBlockstoreID)
+	storageCfg, _ := c.getStorageConfigs()
 	writeResponse(w, r, http.StatusOK, storageCfg)
 }
 
+func (c *Controller) getStorageConfigs() (*apigen.StorageConfig, apigen.StorageConfigList) {
+	storageListCfg := c.getStorageConfigList()
+	if len(storageListCfg) > 1 {
+		// non-empty storage-config-list, return empty storage-config
+		return &apigen.StorageConfig{}, storageListCfg
+	} else {
+		storageCfg, _ := c.getStorageConfig(config.SingleBlockstoreID)
+		return storageCfg, storageListCfg
```

Comment:
We can return an empty configList similar to how we return the storage config in the if clause